vidgear/gears/asyncio/netgear_async.py

```diff
@@ -211,9 +211,13 @@ def __init__(
             # assign local ip address if None
             if address is None:
                 self.__address = "*"  # define address
+            else: 
+                self.__address = address
             # assign default port address if None
             if port is None:
                 self.__port = "5555"
+            else: 
+                self.__port = port
```

Comment:
@otter-in-a-suit add or completely remove `str(port)`. We need consistency in the code.